pkg/urltype/types.go

```diff
@@ -1,16 +1,16 @@
-package url
+package urltype
```

Comment:
We should not put this in a new inner package as this can help get around import cycles and lead to a messed up architecture. This can make the function
```
func someFunction() urltype.URL {return urltype.URL{}}
```
valid in the occlient package which shouldn't be the case as the occlient package shouldn't be aware of the URL related things.